Assignment/Conversion.c

```diff
@@ -0,0 +1,79 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// Conversion.c
+// Program for Number Conversion.
+// ------------------------------------------------------------------------------------------------
+
+#pragma warning(disable:4996)
+#include <stdio.h>
+void DecToBinary (int n);
+void DecToHexa (int n);
+
+// Function to convert the decimal number to binary
+void DecToBinary (int n) {
+   int binary[32];
+   int i = 0;
+   int flag = 0; // Flag is to check if the number is negative
+   if (n == 0)    printf ("binary is 0");
+   if (n < 0) {
+      n = -n;
+      flag = 1;
+   }
+   while (n > 0) {
+      binary[i] = n % 2;
+      n = n / 2;

```

Comment:
Can be written as n/=2 (compound assignment operator). Apply this change wherever possible. Please refer this 
[https://learn.microsoft.com/en-us/cpp/c-language/c-compound-assignment?view=msvc-170](url)